### ISSUE TITLE
`[iZettleSDK presentSettingsFromViewController:]` crashes on iOS 9

### ISSUE THREAD

**Issue by svenmuennich** [iZettle/sdk-ios#313]
# Expected Behavior

Running the following code on iOS 9 opens the settings controller:
```obj-c
[[iZettleSDK shared] presentSettingsFromViewController:myPresentingViewController];
```

# Current Behavior

Running the following code on iOS 9 results in a crash:
```obj-c
[[iZettleSDK shared] presentSettingsFromViewController:myPresentingViewController];
```

The logged exception is:
```
Fatal Exception: NSInvalidArgumentException
-[UIButtonLabel setAdjustsFontForContentSizeCategory:]: unrecognized selector sent to instance
```

# Steps to Reproduce

1. Create an app with a single view controller.
2. Initialize the SDK as documented.
3. Call `[[iZettleSDK shared] presentSettingsFromViewController:self];` after the view controller appeared.
4. Run the app on iOS 9.3.

## Context 

* Operating Version: iOS 9.3.5
* SDK version: 3.0.0

[The documentation of this repository states that the SDK is compatible with iOS 9](https://github.com/iZettle/sdk-ios#requirements). The property [`UILabel. adjustsFontForContentSizeCategory()` (i.e. Objective-C method `[UILabel setAdjustsFontForContentSizeCategory:]`](https://developer.apple.com/documentation/uikit/uicontentsizecategoryadjusting/1771731-adjustsfontforcontentsizecategor), which is called somewhere in `iZettlePayments`, is only available in iOS 10+ though.

## Failure Logs

(inconclusive) stack trace of the crash:
```
Fatal Exception: NSInvalidArgumentException
0  CoreFoundation                 0x20f6f91b __exceptionPreprocess
1  libobjc.A.dylib                0x2070ae17 objc_exception_throw
2  CoreFoundation                 0x20f752b5 __methodDescriptionForSelector
3  CoreFoundation                 0x20f72ee1 ___forwarding___
4  CoreFoundation                 0x20e9e238 _CF_forwarding_prep_0
5  iZettlePayments                0x14eab33 (missing)
6  iZettlePayments                0x14ec3d1 (missing)
7  iZettlePayments                0x14eb2bb (missing)
8  UIKit                          0x2550caf5 -[UIView init]
9  iZettlePayments                0x1540cdd (missing)
10 iZettlePayments                0x14b0617 (missing)
11 UIKit                          0x254ead8b -[UIViewController loadViewIfRequired]
12 UIKit                          0x255a8683 -[UINavigationController _layoutViewController:]
13 UIKit                          0x255a8575 -[UINavigationController _updateScrollViewFromViewController:toViewController:]
14 UIKit                          0x255a7929 -[UINavigationController _startTransition:fromViewController:toViewController:]
15 UIKit                          0x255a755f -[UINavigationController _startDeferredTransitionIfNeeded:]
16 UIKit                          0x2557fbf1 -[UINavigationController pushViewController:transition:forceImmediate:]
17 UIKit                          0x25875485 __54-[UINavigationController pushViewController:animated:]_block_invoke
18 UIKit                          0x258577eb +[UIViewController _performWithoutDeferringTransitions:]
19 UIKit                          0x2557f2db -[UINavigationController pushViewController:animated:]
20 iZettlePayments                0x151dd51 (missing)
21 iZettlePayments                0x14a317d (missing)
22 iZettlePayments                0x151dbdd (missing)
23 iZettlePayments                0x1503365 (missing)
24 iZettlePayments                0x1502dc3 (missing)
25 iZettlePayments                0x1502cf7 (missing)
26 iZettlePayments                0x1501e87 (missing)
27 iZettlePayments                0x150295f (missing)
28 iZettlePayments                0x1503323 (missing)
29 iZettlePayments                0x151da13 (missing)
30 iZettleSDK                     0x1add801 (missing)
31 iZettleSDK                     0x1adebb7 (missing)
32 MyApp                          0xb64cf [...]
```

**Comment by said-iz** [comment]
Hey @svenmuennich, we'll release the fix for this issue in the next SDK release which should come out in the next 2 weeks. However, we strongly recommend to update from iOS 9 and iOS 10 to a newer iOS version since we'll drop support for the older ones in one of the next SDK releases.

Thanks,
Said

**Comment by svenmuennich** [comment]
Thank you for the update.

> [...] we strongly recommend to update from iOS 9 and iOS 10 to a newer iOS version since we'll drop support for the older ones in one of the next SDK releases.

Thanks for the heads up about this! We are in the process of transitioning to a requirement of iOS >= 12 but have to keep an eye on our user base.

**Comment by said-iz** [comment]
Fix released in 3.0.1 release
